README.md

```diff
@@ -72,50 +65,34 @@ import App from 'fusion-react';
 import I18n, {I18nToken, I18nLoaderToken} from 'fusion-plugin-i18n';
 import {FetchToken} from 'fusion-tokens';
 import fetch from 'unfetch';
-import Hello from './hello';
 import I18nLoader from './translations';
 
 export default () => {
   const app = new App(<div></div>);
 
   app.register(I18nToken, I18n);
   __NODE__
-    ? app.configure(I18nLoaderToken, I18nLoader);
-    : app.configure(FetchToken, fetch);
+    ? app.register(I18nLoaderToken, I18nLoader);
+    : app.register(FetchToken, fetch);
 
-  app.register(Hello);
+  // ....
```

Comment:
This change shouldn't remove the part of the example that shows how to actually use the service

Reply:
There are still examples in the readme